console/console-init/ui/src/modules/iot-device-detail/DeviceDetailPage.tsx

```diff
@@ -95,7 +95,7 @@ export default function DeviceDetailPage() {
     () => (
       <Breadcrumb>
         <BreadcrumbItem>
-          <Link id="cdetail-link-home" to={"/"}>
+          <Link id="device-detail-home-link" to={"/"}>
```

Comment:
@ajaypratap003 for being consistent in breadcrumb i suggest to keep the id same at all places in the whole app. WDYT ?

Reply:
@drcoolsanjeev What ID should we keep for all of them?